### PR TITLE
Wolox Footer

### PR DESCRIPTION
**Pull request by tammaroivan**
## Summary
- Added footer

## Screenshot
image

## Trello
https://trello.com/c/a1bnzU0C/14-footer

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -80,5 +80,11 @@ <h1 class="title">Back-End Tools</h1>
   	</div>
   </div>
```
Comment (FrankIglesias) [inline maquetado/index.html]: delete empty line

File: maquetado/index.html [hunk 2]
```diff
@@ -80,5 +80,11 @@ <h1 class="title">Back-End Tools</h1>
...
 
+  <footer>
+    <h2 class="slogan">WE ARE GAME CHANGERS</h2>
+    <img src="assets/wolox_logo_white.svg" alt="">
```
Comment (FrankIglesias) [inline maquetado/index.html]: add alt

File: maquetado/index.html [hunk 3]
```diff
@@ -80,5 +80,11 @@ <h1 class="title">Back-End Tools</h1>
...
+    <p class="copyright">Copyright 2018. All rights reserved</p>
+  </footer>
+
```
Comment (FrankIglesias) [inline maquetado/index.html]: delete line

File: maquetado/scss/footer.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+footer {
```
Comment (FrankIglesias) [inline maquetado/scss/footer.scss]: use a class for this

File: maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 200px;
+  padding: 50px;
+  width: 100%;
+}
+
+.slogan {
+  color: #FFF;
```
Comment (FrankIglesias) [inline maquetado/scss/footer.scss]: use a variable color

File: maquetado/scss/footer.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  font-size: $text-big;
+  letter-spacing: 1px;
+  margin-bottom: 20px;
+}
+
+.copyright {
+  align-self: flex-end;
+  color: #FFF;
```
Comment (FrankIglesias) [inline maquetado/scss/footer.scss]: same

File: maquetado/scss/styles.scss [hunk 7]
```diff
@@ -1,17 +1,19 @@
 $green: #BED23C;
+$blue: #00ADEE;
+
+$text-small: 12px;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: move this to a `font.scss` fille

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: maquetado/scss/navbar.scss [hunk 8]
```diff
@@ -19,5 +19,5 @@
 }
 
 .navbar p {
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: dont use tags in scss

File: maquetado/scss/styles.scss [hunk 9]
```diff
@@ -1,17 +1,15 @@
 $green: #BED23C;
+$blue: #00ADEE;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: move this to colors.scss file

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: maquetado/scss/navbar.scss (hunk 8 above)
Comment (tammaroivan) [reply maquetado/scss/navbar.scss]: Not using tags in previous version (PR Fixes in navbar pr)

File: maquetado/scss/styles.scss (hunk 9 above)
Comment (tammaroivan) [reply maquetado/scss/styles.scss]: moved

File: maquetado/scss/styles.scss [hunk 10]
```diff
@@ -1,16 +1,12 @@
...
 }
 
-body {
-	padding: 100px;
-}
-
 .main {
 	width: 70%;
 	margin: 0 auto;
+	margin-top: 70px;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: check indentation

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: maquetado/scss/styles.scss [hunk 11]
```diff
@@ -1,21 +1,17 @@
...
 }
 
 .cards {
-	display: flex;
-	flex-wrap: wrap;
-	margin: 50px;
-	margin-top: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 50px;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: use shorthand here

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: maquetado/scss/styles.scss [hunk 12]
```diff
@@ -1,21 +1,15 @@
...
-body {
-	padding: 100px;
+  font-family: 'Roboto', sans-serif;
 }
 
 .main {
-	width: 70%;
-	margin: 0 auto;
+  width: 70%;
+  margin: 70px 0 auto;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: why this auto?

File: maquetado/scss/styles.scss [hunk 13]
```diff
@@ -1,21 +1,15 @@
...
-
-body {
-	padding: 100px;
+  font-family: 'Roboto', sans-serif;
 }
 
 .main {
-	width: 70%;
-	margin: 0 auto;
+  width: 70%;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: use a fix width instead

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**